ERROR - 2023-11-03 22:23:43 --> mysqli_sql_exception: Unknown column 'email' in 'where clause' in C:\xampp\htdocs\proyectopauny\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\proyectopauny\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\proyectopauny\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\proyectopauny\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\proyectopauny\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyectopauny\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyectopauny\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('facundo@gmail.c...', 'email', Array, NULL, 'correo_electron...')
#6 C:\xampp\htdocs\proyectopauny\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('correo_electron...', 'correo_electron...', 'facundo@gmail.c...', Array, Array)
#7 C:\xampp\htdocs\proyectopauny\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\proyectopauny\app\Controllers\usuario_controller.php(23): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\proyectopauny\system\CodeIgniter.php(919): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\proyectopauny\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\proyectopauny\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\proyectopauny\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 22:23:43 --> Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('facundo@gmail.com', 'email', [...], null, 'correo_electronico')
 4 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('correo_electronico', 'correo_electronico', 'facundo@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(23): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-11-03 22:26:47 --> mysqli_sql_exception: Unknown column 'correo_electronico' in 'field list' in C:\xampp\htdocs\proyectopauny\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\proyectopauny\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\proyectopauny\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\proyectopauny\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\proyectopauny\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\proyectopauny\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectopauny\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectopauny\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\proyectopauny\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\proyectopauny\app\Controllers\usuario_controller.php(42): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\proyectopauny\system\CodeIgniter.php(919): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\proyectopauny\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\proyectopauny\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\proyectopauny\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 22:26:47 --> Unknown column 'correo_electronico' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuario`, `correo_electronico`, `pass`) VALUES (:usuario:, :correo_electronico:, :pass:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\usuario_controller.php(42): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
